src/pkg/projects/Microsoft.NETCore.App/src/Microsoft.NETCore.App.depproj

```diff
@@ -36,8 +36,8 @@
     </ItemGroup>
 
     <PropertyGroup>
-      <_crossDir Condition="'$(NuGetRuntimeIdentifier)' == 'win8-arm'">/x86_arm</_crossDir>
-      <_crossDir Condition="'$(NuGetRuntimeIdentifier)' == 'win10-arm64'">/x64_arm64</_crossDir>
+      <_crossDir Condition="'$(NuGetRuntimeIdentifier)' == 'win8-arm' or '$(NuGetRuntimeIdentifier)' == 'win-arm'">/x86_arm</_crossDir>
```

Comment:
This could also be written as (untested)

```
<_crossDir Condition="$(NugetRuntimeIdentifier.EndsWith('-arm'))">x86_arm</_crossDir>
<_crossDir Condition="$(NugetRuntimeIdentifier.EndsWith('-arm64'))">x64_arm64</_crossDir>
```